src/local.rs

```diff
@@ -103,6 +103,10 @@ where
     task.build_dag(&mut eval, &eval_config)
         .context("Cannot build the task DAG")?;
 
+    if let Some(priority) = opt.priority {
```

Comment:
I'm not sure I 100% like this here. There is [ExecutionDAGConfig](https://github.com/edomora97/task-maker-rust/blob/348d1ed9d8275eea8b0d8b32e4c11014d2594963/task-maker-dag/src/dag.rs#L27) which sounds to me like a better place for this kind of configurations.

Reply:
I thought about it, but decided against it because ExecutionDAGConfig influences *how* the DAG gets executed, and priority doesn't.